docs/NMOS With H.264.md

```diff
@@ -138,32 +181,228 @@ The SDP file at the `manifest_href` MUST comply with the requirements of RFC 618
+
+Informative note: The `static` mode allows multiple SPS and PPS to be used by the coded bitstream as long as the active values associated with Flow attributes do not change. For example scaling list are allowed to change when the profile allows it.
+
+A Receiver with this capability set to `dynamic` supports that a coded stream uses parameter sets that MAY be associated with multiple Flows. Such parameter sets MAY be obtained out-of-band or in-band according to the Receiver `parameter_sets_transport_mode` Capability. When obtained out-of-band the `sprop-parameter-sets` parameter of an SDP transport file or the parameter sets from a transport specific out-of-band mechanism MAY contain parameter sets associated with multiple Flows. When obtained in-band the `sprop-parameter-sets` parameter of an SDP transport file or the parameter sets from a transport specific out-of-band mechanism MUST be empty or omitted and the Sender MAY transmit in-band parameter sets to update parameter sets initially received out-of-band or to add and update new ones. At all time, the Sender is allowed to transmit in-band parameter sets that are duplicates of the parameter sets obtained either out-of-band or in-band. When obtained in-and-out-of-band, in-band parameter sets have priority over out-of-band parameters sets.
+
+### Senders
+
+A Sender is allowed, unless constrained by IS-11, to produces an H.264 coded stream that is compliant with the `profile` and `level` of the associated Flow. Such a Sender MAY use one or multiple active parameter sets as per the [H.264][] specification. A Sender MAY seamlessly change dynamically the coded stream's active parameter sets, provided that the Flow associated with the Sender changes accordingly and the content of the SDP transport file does not change. If the content of the SDP transport file changes, the Sender MUST comply with IS-04, IS-05. A Sender indicates its mode of operation with the `parameter_sets_flow_mode` and `parameter_sets_transport_mode` attributes.
```

Comment:
I'm a bit confused...
"Allowed" or "REQUIRED"?
And even if constrained by IS-11, the stream MUST be compliant with the `profile` and `level`?

Reply:
will be adding "any" ... "A Sender is allowed, unless constrained by IS-11, to produces any H.264 coded stream ..." and change is allowed for MAY ... There is already normative text requiring an encoder to produce a stream that is compliant with the profile and level.